src/endpoints/createFeedbackEndpoint.js

```diff
@@ -46,16 +46,16 @@ export default (baseUrl: string): Endpoint<ParamsType, {}> => new EndpointBuilde
   .withParamsToBodyMapper((params: ParamsType): FormData => {
     const formData = new FormData()
     formData.append('rating', params.isPositiveRating ? POSITIVE_RATING : NEGATIVE_RATING)
-    if (params.permalink) {
+    if (params.permalink !== undefined) {
```

Comment:
I think this check should be `params.permalink !== undefined && params.permalink !== null`. What happens if null is sent to the cms?

@timoludwig

Reply:
It makes no difference for the cms whether the permalink parameter is `null` or just not sent at all.

Reply:
Also, `null` is not allowed by flow

Reply:
Also, null is not an allowed type by flow.